elements/pf-dropdown/pf-dropdown-group.ts

```diff
@@ -15,17 +16,60 @@ export class PfDropdownGroup extends LitElement {
 
   static override readonly shadowRootOptions: ShadowRootInit = { ...LitElement.shadowRootOptions, delegatesFocus: true };
 
+  /**
+   * Disable the dropdown
+   */
+  @property({ type: Boolean, reflect: true }) disabled = false;
+
```

Comment:
this doesn't appear on v4's group, so i wonder if we should use a context for this instead? or perhaps just a query selector?